src/install/ykfde

```diff
@@ -22,6 +22,26 @@ build() {
     add_all_modules "/crypto/"
   fi
 
+  base_files=$(pacman -Ql python | awk '{print $2}' | /bin/grep -E '.py$|.so$')
+  for file in $base_files
+  do
+    add_file $file
+  done
+
+  python_ver=$(python -V | awk '{print $2}')
+  add_full_dir "/usr/lib/python${python_ver::-2}/site-packages/zipp"
+  add_full_dir "/usr/lib/python${python_ver::-2}/site-packages/cryptography"
+  add_full_dir "/usr/lib/python${python_ver::-2}/site-packages/cffi"
+  add_full_dir "/usr/lib/python${python_ver::-2}/site-packages/smartcard"
+  add_full_dir "/usr/lib/python${python_ver::-2}/site-packages/fido2"
+  add_full_dir "/usr/lib/python${python_ver::-2}/site-packages/importlib_metadata"
+  add_full_dir "/usr/lib/python${python_ver::-2}/site-packages/ykman"
+  add_full_dir "/usr/lib/python${python_ver::-2}/site-packages/yubikit"
+  add_full_dir "/usr/lib/python${python_ver::-2}/site-packages/click"
+  add_full_dir "/usr/lib/python${python_ver::-2}/site-packages/keyring"
+  add_binary "/usr/lib/python${python_ver::-2}/site-packages/_cffi_backend.cpython-311-x86_64-linux-gnu.so"
```

Comment:
Hardcoding python version is sensitive to python updates.

In general we're exposed to potential future dependency changes of yubikey-manager which makes me worried a bit.

Reply:
on Archlinux, it's normal version, because the path includes version. the problem is if version like: 3.11.10 and above will break the script. I want to parse like: python -V | awk '{print $2}' | awk -F'.' '{print $1"."$2}' for exact version, but I believe that it cannot be .10.
In general, we need unit test for initramfs, the package must pass the test before upgrading version. I will write it in this week.

Reply:
my bad, I did not see this. wait a minute.

Reply:
I just pushed new commit to correct it. Thank you very much.

Reply:
Unit test, I mean CI, yeah, I understand your concer, but we must test it before release, right?